src/index.ts

```diff
@@ -45,3 +45,4 @@ export {
   transformSource, // Not used here but adds only 222 uncompressed (60 compressed) bytes of code
   UnknownSources,
 } from './utils/entropy_source'
+export { withIframe } from './utils/dom'
```

Comment:
Please add a warning comment used with every other undocumented export

Reply:
It already is, my bad

Reply:
I added it below the warning comment above, which I'd assume includes this export too. Should I add another one regardless, to be sure?

Reply:
I mean this message that you can see in IDE or in `fp.d.ts`:

<img width="807" alt="image" src="https://github.com/fingerprintjs/fingerprintjs/assets/9006227/8feac2ce-0a52-4e51-8056-3ad9670f6e90">

It's added right above the function declaration.